src/features/MoviesList/MovieDetails/index.js

```diff
@@ -15,7 +15,7 @@ export const MovieDetails = () => {
         if (id) {
             dispatch(fetchMovieDetails(id))
         }
-    }, []);
+    }, [dispatch]);
 
```

Comment:
Is it necessary to insert "dispatch" if we have in the code "const dispatch= useDispatch()" in line 9?